src/authInfo.ts

```diff
@@ -198,42 +198,6 @@ export enum SfdcUrl {
   PRODUCTION = 'https://login.salesforce.com',
 }
 
-const INTERNAL_URL_PARTS = [
-  '.internal.',
-  '.vpod.',
-  'stm.salesforce.com',
-  '.blitz.salesforce.com',
-  'mobile1.t.salesforce.com',
-];
-
-function isInternalUrl(loginUrl = ''): boolean {
-  return loginUrl.startsWith('https://gs1.') || INTERNAL_URL_PARTS.some((part) => loginUrl.includes(part));
-}
-
-function getJwtAudienceUrl(options: OAuth2Options) {
-  // default audience must be...
-  let audienceUrl: string = SfdcUrl.PRODUCTION;
-  const loginUrl = getString(options, 'loginUrl', '');
-  const createdOrgInstance = getString(options, 'createdOrgInstance', '').trim().toLowerCase();
```

Comment:
createdOrgInstance isn't even a member of `OAuth2Options`.  Using this getString seems to defeat the purpose of typescript.

Reply:
Ohh it is for sure getting past typescript =P But the majority of this code existed before typescript and I don't think we wanted to update the jsforce types since it is specific to us. Granted, we should have fixed the typing in this method and added a comment.

This is needed because of scratch org creations. 

In the header, you can change it too `function getJwtAudienceUrl(options: OAuth2Options & { createdOrgInstance?: string }) {` if you want to fix the typing.